### PR TITLE
excluded superclasses that have MONDO-XXXXXXX-obsoleted source

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #6004

@sabrinatoro

- I will slowly work on cleaning up these superclasses that were rewired
- If I disagree with the subclass, I made it an excluded subclass with my ORCID
- if I agree with the superclass assertion, I added my ORCID
- I cleaned up a few definitions too

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -133834,15 +133835,15 @@ xref: GARD:17843 {source="Orphanet:468666"}
 xref: OMIM:106190 {source="Orphanet:468666", source="MONDO:equivalentTo", source="DOID:0060603"}
 xref: Orphanet:468666 {source="MONDO:equivalentTo"}
 xref: UMLS:C5568836 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0005093 {source="MONDO:0018798-obsoleted"} ! skin disorder
 is_a: MONDO:0006527 {source="DOID:0060603", source="MONDO:Entailed", source="MONDO:Redundant"} ! anhidrosis
 is_a: MONDO:0019296 {source="Orphanet:468666"} ! subcutaneous tissue disorder
 is_a: MONDO:0021154 {source="Orphanet:468666"} ! dermis disorder
 intersection_of: MONDO:0006527 ! anhidrosis
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6181 ! ITPR2
+relationship: excluded_subClassOf MONDO:0005093 {source="MONDO:0018798-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! skin disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:133843]: another parent is "dermis disorder", which is a "skin disorder". So excluded this SC does not make sense. 

Also this term is a "anhidrosis" (= sweat gland disorder) "with normal sweat gland". 

This term requires more review, but at minimum, we should either not exclude SC "skin disorder", or also exclude SC "dermis disorder"

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -135083,12 +135084,12 @@ xref: OMIM:108900 {source="MONDO:equivalentTo", source="Orphanet:1479", source="
 xref: Orphanet:1479 {source="MONDO:equivalentTo", source="OMIM:108900", source="DOID:0110112"}
 xref: UMLS:C1862388 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C3276096 {source="MONDO:equivalentTo", source="OMIM:108900", source="MONDO:MEDGEN", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0005267 {source="MONDO:0015506-obsoleted"} ! heart disorder
 is_a: MONDO:0006664 {source="DC-OMIM:108900", source="DOID:0110112", source="MONDO:Redundant", source="OMIM:108900", source="OMIM:108900/inferred"} ! atrial septal defect
 intersection_of: MONDO:0006664 ! atrial septal defect
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2488 ! NKX2-5
+relationship: excluded_subClassOf MONDO:0005267 {source="MONDO:0015506-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! heart disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:135090]: 'heart disorder' is a great-grandparent. We can remove this parent, but we should not add as an "ecluded_SC"

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -139282,9 +139286,10 @@ xref: UMLS:C1838416 {source="OMIM:119600", source="MONDO:notFoundInDiseaseSubset
 xref: UMLS:C1861516 {source="OMIM:119600", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0002254 {source="NCIT:C75020"} ! syndromic disease
 is_a: MONDO:0005516 {source="DOID:13994", source="MESH:D002973"} ! osteochondrodysplasia
-is_a: MONDO:0018230 {source="MONDO:0019709-obsoleted", source="https://github.com/monarch-initiative/mondo/pull/2571/"} ! skeletal dysplasia
+relationship: excluded_subClassOf MONDO:0018230 {source="MONDO:0019709-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! skeletal dysplasia
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:139289]: I would remove but not "exclude_SC"
The parent "osteochondrodysplasia" seems that is should be "skeletal dysplasia" (though it currently is not).

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky 
There are a few examples where I think "exclude_subclassOf" is incorrect because it would be excluding a grandparent term. 

In other cases, I "think" it is ok, but I am not an expert. Therefore I am wondering whether it wouldn't be better to remove these subClassOf (instead of excluding them), so we don't restrict future potential parentage. I will add this specific point to be discussed on a curation call.

**Comment by nicolevasilevsky** [comment]
We can close this, we'll address this with the action items in the ticket (https://github.com/monarch-initiative/mondo/issues/6004#issuecomment-2038336961).

**Pull request closed without merging**